src/main/java/seedu/duke/commands/HelpCommand.java

```diff
@@ -24,14 +23,12 @@ public HelpCommand() {
     @Override
     public void execute(Ui ui) {
         ui.printWelcomeTutorial();
-        String response;
         String command;
         boolean isNotDone = true;
 
         while (isNotDone) {
-            System.out.println(ui.INSTRUCTIONS[numOfCommandsDone]);
-            response = SC.nextLine();
-            command = response.trim().toLowerCase();
+            ui.printInstruction(numOfCommandsDone);
+            command = ui.readLine();
             try {
```

Comment:
Oh this `try-catch` block is probably not needed since all `DukeException` are caught in the main `Duke` class right?

Reply:
Then this thread will terminate execution and jump out of the help mode. ^_^'

Reply:
Alright noted then, it's like a mini instance of `Duke`.